### ISSUE TITLE
sliding is choppy

### ISSUE THREAD

**Issue by Duttbust** [Another-Axiom/GorillaLocomotion#6]
the "surface" script works, but is very broken and choppy. how could that be fixed

**Comment by wndalci** [comment]
Any update on this? I can't get it to work normally at all :(

Ive tried numbers between 1-100 and also 0-1 but nothing works. It just glitches weirdly with no sliding

**Comment by Duttbust** [comment]
no, nothing that ive found. sorry :(

**Comment by TheScruffyKat** [comment]
I'll try and remake the sliding for my "GorillaMotion" update to it where I fix and add things. I'll update you when its ready

**Comment by TheScruffyKat** [comment]
About 40% through now as I had a massive break and had to redo some stuff

**Comment by sunny-dee-71** [comment]
yeah found the problem it is to do with the sliding only being updated when you first collide not after i am working on a fix tho

**Comment by Cdev-git** [comment]
Try adding a physics mat with no friction to the slippery obj along with the surface script.
